helix-rest/src/test/java/org/apache/helix/rest/metadatastore/accessor/TestZkRoutingDataWriter.java

```diff
@@ -132,61 +132,86 @@ public void testSetRoutingData() {
   public void testAddMetadataStoreRealmNonLeader() {
     MockWriter mockWriter = new MockWriter(DUMMY_NAMESPACE, ZK_ADDR);
     mockWriter.addMetadataStoreRealm(DUMMY_REALM);
-    Assert.assertEquals("PUT", mockWriter.calledRequest.getMethod());
+    Assert.assertEquals(mockWriter.calledRequest.getMethod(), "PUT");
```

Comment:
Could we use a pre-defined constant instead of "PUT"? (Cue: HttpVerbs..) Same applies below.

Reply:
Sure.